File: python/python_fundamentals/functions/functions_exercise/loading_bar.py
def loading_bar(number):
    loaded = ["["]
    index_number = number // 10

    for idx in range(1, 11):

        if idx <= index_number:
            loaded.insert(idx, "%")
        else:
            loaded.insert(idx, ".")

        if idx == 10:
            loaded.append("]")

    if index_number < 10:
        return f"{number}% {''.join(loaded)}\n"\
               f"Still loading..."
    else:
        return f"100% Complete!\n"\
               f"{''.join(loaded)}"

some_numer = int(input())
print(loading_bar(some_numer))

File: Front-End JS/JS Front-End/FunctionsAndStatementsExercises/LoadingBar.js
function loadingBar(percent) {
    theBar = [".",".",".",".",".",".",".",".",".","."]
    progress = percent / 10
    
    if (percent == 100){
        theBar = ["%","%","%","%","%","%","%","%","%","%"]
        console.log("100% Complete!");
        console.log(`[${theBar.join("")}]`)

    }
    else {
        for (let i = 0; i < progress; i++){
            theBar[i] = "%"
        }
        console.log(`${percent}% [${theBar.join("")}]`)
        console.log("Still loading...");

    }



}

loadingBar(30)